### DISCUSSION TITLE
Styling on input not possible

### DISCUSSION THREAD

**Discussion by ekandreas** [filamentphp/filament#1021] [General]
Since v2.6.1 it is no longer possible to style the input fields.
We are not using a rounded styling so we need to manipulate the input class field.
When releasing the v2.6.1 extraAttributes has been moved up one node in the DOM so it isn't possible to manipulate the input field. Eg, 
```
TextInput::make('sort_order')
            ->label(__('groups.sort_order'))
            ->required()
            ->numeric()
            ->extraAttributes([
                'class' => 'border rounded-none pl-2',
            ]);
```

Can we have some more prop to set the input field in any way?

**Comment by danharrin** [comment]
Check out my comment on the PR :)
